src/app/components/LiveLabel/index.tsx

```diff
@@ -0,0 +1,50 @@
+/** @jsx jsx */
+/** @jsxRuntime classic */
+import { PropsWithChildren, useContext } from 'react';
+import { jsx } from '@emotion/react';
+import VisuallyHiddenText from '../VisuallyHiddenText';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import styles from './index.styles';
+
+interface LiveLabelProps {
+  ariaHidden?: boolean;
+  liveText?: string;
+  offScreenText?: string;
+  lang?: string;
+  id?: string;
+}
+
+const LiveLabel = ({
+  ariaHidden = false,
+  liveText = 'LIVE',
+  offScreenText = '',
+  lang = 'en-GB',
+  id,
+  children,
+}: PropsWithChildren<LiveLabelProps>) => {
+  const { dir } = useContext(ServiceContext);
+  const isRtl = dir === 'rtl';
+
+  return (
+    // lines 27, 56,66, 31 concerning with id are a temporary fix for the a11y nested span's bug experienced in TalkBack, refer to the following issue: https://github.com/bbc/simorgh/issues/9652
+    // eslint-disable-next-line jsx-a11y/aria-role
+    <span role="text" id={id}>
+      <span
```

Comment:
Could the `<Text as='span'>` component be used here along with the `fontVariant='sansBold'` prop? Would allow you to remove the manual bold styling in the CSS stylesheet.

Reply:
Oooh that might be worth a try 👀 Thanks for the suggestion

Reply:
@amoore108 We've implemented the change and are now just checking out the consequent failing snapshots. We think these changes (screenshot) are a result of using the fontVariant prop via the Text component, but aren't too sure if a) that's correct and b) if they are safe to update? 

<img width="844" alt="Screenshot 2023-10-12 at 10 51 24" src="https://github.com/bbc/simorgh/assets/120198886/915d045e-4b6f-4c09-aa16-22dd73a85bf1">

Reply:
The `<Text>` component defaults to `size = 'pica',`, which is why its included some additional media queries to handle the sizing on different breakpoints. This tends to follow GEL breakpoints, so maybe the `size` prop also needs to be set to the correct size for the LiveLabel based on the designs?

Reply:
Ah okay, thank you!